### PR TITLE
Allow PHP 8 installation

### PR DESCRIPTION
**Pull request by ubkpesek**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
       "issues": "https://github.com/omines/datatables-bundle/issues"
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.2|^8.0",
```
Comment (maxhelias) [inline composer.json:23]: We should use `>=7.2`

Comment (ubkpesek) [reply composer.json]: I have updated it.

**Comment by curry684** [comment]
Please also add it to `.travis.yml` now that Travis indeed supports it (this is why I didn't do it before yet). If support is formally declared in `composer.json` we must test it.

**Comment by ubkpesek** [comment]
> 
> 
> Please also add it to `.travis.yml` now that Travis indeed supports it (this is why I didn't do it before yet). If support is formally declared in `composer.json` we must test it.

I have never worked with Travis before and I am not sure which config I should add so I just copied PHP 7.4. Not sure why there are eg. dev dependcies needed.

**Comment by curry684** [comment]
Fine like this, thanks   :)

**Pull request merged**